### PR TITLE
Change config BigFileThreshold default to 1MB

### PR DESCRIPTION
**Pull request by chenji-kael**
Place active appendable file to rabbit at first. When it get colder ,it will
be move to tiger

Signed-off-by: chenji <insomnia@139.com>

### CODE REVIEW

File: meta/client/tidbclient/object.go [hunk 1]
```diff
@@ -296,6 +296,121 @@ func (t *TidbClient) ReplaceObjectMetas(object *Object, tx Tx) (err error) {
...
+	if bMarker == "" && oMarker == "" && vMarker == "" {
+		sqltext = "select bucketname,name,version,location,pool,ownerid,size,objectid,lastmodifiedtime,etag,contenttype," +
+			"customattributes,acl,nullversion,deletemarker,ssetype,encryptionkey,initializationvector,type,storageclass,createtime" +
+			" from hotobjects order by bucketname,name,version limit ?;"
+		helper.Logger.Info("sqltext0", sqltext)
+		rows, err = t.Client.Query(sqltext, limit)
+	} else {
+		sqltext = "select bucketname,name,version,location,pool,ownerid,size,objectid,lastmodifiedtime,etag,contenttype," +
+			"customattributes,acl,nullversion,deletemarker,ssetype,encryptionkey,initializationvector,type,storageclass,createtime" +
+			" from hotobjects where bucketname>=? and name >=? and version >=? order by bucketname,name,version limit ?;"
```
Comment (cannium) [inline meta/client/tidbclient/object.go:343]:
```
A_bucket_A_object_A_version
A_bucket_A_object_B_version <- cursor
A_bucket_B_object_A_version
```
Suppose cursor is at `A_bucket_A_object_B_version`, `A_bucket_B_object_A_version` would be skipped.

Comment (cxt90730) [reply meta/client/tidbclient/object.go:343]: In the previous design, the first row should be `A_bucket_A_object_B_version` after next query, and this row would be skiped.

File: meta/types/object.go [hunk 2]
```diff
@@ -45,6 +45,13 @@ type Object struct {
 	CreateTime   uint64 // Timestamp(nanosecond)
 }
 
+type ScanHotObjectsResult struct {
+	NextBMarker string
+	NextOMarker string
+	NextVMarker string
+	Objects     []Object
+}
+
```
Comment (cannium) [inline meta/types/object.go:54]: Not necessary any more.

File: storage/object.go [hunk 3]
```diff
@@ -41,21 +41,87 @@ func (yig *YigStorage) pickRandomCluster() (cluster backend.Cluster) {
 	return
 }
 
+func (yig *YigStorage) PickSpecificCluster(poolName string) (cluster backend.Cluster) {
```
Comment (cannium) [inline storage/object.go:44]: If only need to pick a specific cluster, why weight calculation is needed?

Comment (chenji-kael) [reply storage/object.go:44]: maybe not only one ssd cluster

**Pull request merged**